### DISCUSSION TITLE
Difference between Conference and Journal Submissions (BraTS)

### DISCUSSION THREAD

**Discussion by stalhabukhari** [MIC-DKFZ/nnUNet#501] [Q&A] [answered]
Hi @FabianIsensee 

I hope you're doing great. I read your [conference paper](https://arxiv.org/abs/1809.10483) on Brain Tumor Segmentation, and recently read the [journal version](https://www.nature.com/articles/s41592-020-01008-z) that discusses other applications as well. I wish to inquire specifically about Brain Tumor Segmentation.

I noticed that the nnUNet reported in the journal version is slightly different from the one reported in the BraTS 2018 challenge. Specifically, the differences included the use of Leaky-ReLUs instead of ReLUs, use of auxiliary loss functions for deep supervision, the absence of any mention of region-based prediction (similar to the cascade of [Wang et al. (2017)](https://arxiv.org/abs/1709.00382)), and the use of SGD + Nesterov Momentum (with a per-epoch decaying schedule for learning rate) instead of Adam optimizer with constant learning rate. Kindly correct me if I missed something.

Are the results/plots for Brain Tumor Segmentation shown in the journal version different from the conference one? If so, then which version would you prefer *specifically* for Brain Tumor Segmentation? Was any comparison done in this regard?

Best Regards,
Talha

**Comment by FabianIsensee** [comment, marked as answer]
Hi,
the arxiv paper you are referencing is NOT nnU-Net. It is a separate project that happens to have a badly chosen title ;-)
For brain tumor segmentation on the BraTS datset I would stick to our BraTS 2020 implementation which is based on nnU-Net: https://arxiv.org/abs/2011.00848. Code for that is in the nnunet repository. If you are using a different dataset (and a non-region based evaluation) then using the standard nnU-Net is best
Best,
Fabian

EDIT: I changed the word 'jounal' -> 'arxiv' in the first sentence. That was a mistake. Now everything is correct: The "No New Net" arxiv paper is NOT nnU-Net! The Nature Methods journal paper IS nnU-Net :-)

**Reply by stalhabukhari** [reply]
> the journal paper you are referencing is NOT nnU-Net. It is a separate project that happens to have a badly chosen title ;-)

Oh! Then how should I interpret the results for brain tumor segmentation in the journal paper? :D

> For brain tumor segmentation on the BraTS datset I would stick to our BraTS 2020 implementation which is based on nnU-Net: https://arxiv.org/abs/2011.00848.

Thank you for the clarification. I'll go through it in detail and will let you know then.

> Code for that is in the nnunet repository.

But the journal paper points to *this* repository under the section **Code availability**. Should I refer to different repositories for the papers I mentioned?

> If you are using a different dataset (and a non-region based evaluation) then using the standard nnU-Net is best

Alright!

**Reply by FabianIsensee** [reply]
The Nature Methods paper only uses Task01 from the MSD for brain tumor segmentation. That is all the old (and not very well annotated) BraTS data. I would recommend to stick to our 2020 arxiv paper for BraTS-like brain tumor segmentation and for all other brain tumor segmentation just apply the nnU-Net from the journal paper.

This repository is the code for the journal paper. Within this repository is a small section that covers the BraTS 2020 - specific adaptations that we made. 
You can find them here: https://github.com/MIC-DKFZ/nnUNet/tree/master/nnunet/training/network_training/competitions_with_custom_Trainers

And here: https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunet/dataset_conversion/Task082_BraTS_2020.py

**Reply by FabianIsensee** [reply]
Ah shoot. I meant to say that the BraTS 2018 arxiv paper "No New Net" is NOT nnU-Net! haha. That was something else that was just named badly.

The journal paper of course is. This repository is for the journal paper.

**Reply by stalhabukhari** [reply]
> Ah shoot. I meant to say that the BraTS 2018 arxiv paper "No New Net" is NOT nnU-Net! haha. That was something else that was just named badly.

I thought so! Okay, thank you for your time!
